### DISCUSSION TITLE
How could I help translate ?

### DISCUSSION THREAD

**Discussion by username4447** [PrincessRTFM/XIVComboPlugin#49] [Q&A]
Ello, I'd like to help translate the plugin in French, how does that work ?

**Comment by VariableVixen** [comment]
It's a work in progress on my end right now, I'm afraid. I'm trying to figure out how to handle multiple languages and translation in the plugin itself.

I'm very sorry about the delay, but I can let you know once the plugin is ready for actual translation work if you still want to help!
